app/models/fee/fixed_fee.rb

```diff
@@ -0,0 +1,27 @@
+# == Schema Information
+#
+# Table name: fees
+#
+#  id          :integer          not null, primary key
+#  claim_id    :integer
+#  fee_type_id :integer
+#  quantity    :integer
+#  amount      :decimal(, )
+#  created_at  :datetime
+#  updated_at  :datetime
+#  uuid        :uuid
+#  rate        :decimal(, )
+#  type        :string
+#
+
+class Fee::FixedFee < Fee::BaseFee
+
+  belongs_to :fee_type, class_name: Fee::FixedFeeType
+
+  validates_with Fee::FixedFeeValidator
+
+  def is_fixed?
+    true
```

Comment:
Could this not be put on the BaseFee as a define_method?

```
define_method "is_#{self.class.to_s.underscore.split('_').first}?" do
  true
end
```

This would add the #is_fixed? #is_misc? methods to inheriting fee subclasses. Just a thought.